deploy/helm/grafana-operator/templates/servicemonitor.yaml

```
@@ -29,11 +29,11 @@ spec:
       {{- end }}
       {{- if .Values.serviceMonitor.metricRelabelings }}
       metricRelabelings:
-      {{ toYaml .Values.serviceMonitor.metricRelabelings | indent 4 }}
+      {{- toYaml .Values.serviceMonitor.metricRelabelings | indent 6 }}
```

```suggestion
      {{- toYaml .Values.serviceMonitor.metricRelabelings | nindent 8 }}
```
This part of the template is not working properly (try to run `helm template` while passing non-empty `metricRelabelings`). To fix that, you can just switch from `indent` to `nindent`.

As a side change, I think it's better to increase indentation to 8 to improve readability.

---

i make a change and add the output of manifest